### ISSUE TITLE
DLL problem with Hannover on 64 bit Windows 7 Home Premium

### ISSUE THREAD

**Issue by Michael-CAP** [qgis/QGIS#41921]
My operating system is 64 bit Windows 7 Home Premium. I downloaded QGIS (Hannover release) successfully, but when I tried to run it, I got the error message 
"The program can't start because api-crt-runtime-I1-1-0.dll is missing from your computer. Try reinstalling the program to fix this problem".  
So I uninstalled QGIS and reinstalled it on another day.  
Got the same result.  
I have had this issue once in a while with some downloads in the past, and all 
the remedies I could find via internet searches did not work.  
Someone suggested the following:
Windows + X keys and select Windows PowerShell (Administrator), in the window that will open, 
you must type the commands in their respective order: 
• DISM.exe /Online /Cleanup-image /Scanhealth 
• DISM.exe /Online /Cleanup-image /Restorehealth 
• DISM.exe /online /cleanup-image /startcomponentcleanup 
• sfc /scanno
My system did not recognize the syntax of the commands. 
I did word searches for the commands provided and found a command that my system recognized: 
SFC /scannow, so I executed that.  It scanned the system and found no problems.  
So I was right back to doing internet searches.  
I tried requesting a Windows 7 upgrade from my system, but got a message saying that Microsoft no longer supports it.  
One of the proposed solutions on line was to download Visual C++ 2015 Redistributable, so I tried that, and it wouldn't install.  
I noticed that what was already on my system was Visual C++ 2008 Redistributable.  
Any ideas?

**Comment by gioman** [comment]
@Michael-CAP seems pretty much a local issue. For sure is not a general one.

**Comment by Michael-CAP** [comment]
I have downloaded a copy of the dll and pasted it into my Windows/System 32 folder.

Still got the same result.

Tried to register the file, but got an error message "The module "api-ms-win-crt-runtime-|1-1-0.dll" was loaded but the entry-point DllRegisterServer was not found.  Make sure that "api-ms-win-crt-runtime-|1-1-0.dll" is a valid DLL or OCX file and then try again"

So... every remedy I have found on line has not fixed the problem.

I am at a loss as to how to proceed.
